services/QuillLMS/spec/services/vitally_integration/serialize_vitally_sales_account_spec.rb

```diff
@@ -84,7 +84,7 @@
       frl: 0,
       ppin: nil,
       nces_id: '111111111',
-      school_subscription: 'NA',
+      school_subscription: 'N/A',
```

Comment:
How about persisting this string literal in a constant, since it is used so much?